### PR TITLE
Fixes #80: Makes absolute positioning CSS explicit so that RTL can't change them.

### LINKED ISSUE #80: toggle button wrongly positioned in RTL
Currently, [the CSS](https://github.com/PolymerElements/paper-toggle-button/blob/c76f64d6d77d591b21a378a75fc79d572fd8cf72/paper-toggle-button.html#L122) to move `div#toggleButton` is set via:

``` css
:host([checked]) .toggle-button {
  transform: translate(16px, 0);
}
```

CSS does not flip coordinates based on directionality, so in RTL this `16px` should be flipped to `-16px`.

Here's an example of what `<paper-toggle-button>` looks like in Chrome's settings page:

top is unfixed, bottom is fixed, whole page is in RTL (i.e. `<html dir=rtl>`)

/cc @tjsavage @notwaldorf

### PR DESCRIPTION
**Pull request by bicknellr**
No flipped on / off sides like Android, however. That'll take a bit more work given that the dragging logic assumes LTR. (Fixes #80.)

### CODE REVIEW

File: paper-toggle-button.html [hunk 1]
```diff
@@ -136,6 +137,8 @@
         position: absolute;
         top: -14px;
         left: -14px;
+        right: unset;
```
Comment (notwaldorf) [inline paper-toggle-button.html]: I don't think this is supported in any IE: http://caniuse.com/#search=unset
Can you use `auto` instead?

Comment (bicknellr) [inline paper-toggle-button.html]: Changed.

**Comment by bicknellr** [comment]
ping

**Comment by notwaldorf** [comment]
LGTM

**Pull request merged**